test/e2e/suite_test.go

```diff
@@ -109,7 +110,15 @@ var _ = ginkgo.BeforeSuite(func() {
 
 	nodeid := os.Getenv("nodeid")
 	kubeconfig := os.Getenv(kubeconfigEnvVar)
-	blobDriver = blob.NewDriver(nodeid)
+	value, exists := os.LookupEnv("ENABLE_BLOBFUSE_PROXY")
+	useBlobfuseProxy := false
+	if exists {
+		useBlobfuseProxy, err = strconv.ParseBool(value)
+		if err != nil {
+			log.Println("Failed to parse boolean value", err)
```

Comment:
no need to parse boolean, if this ENV is defined, then use blobfuse-proxy, and I took a look at  logs, looks like it's still not using proxy: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_blob-csi-driver/349/Failed%20to%20parse%20boolean%20value/1367741062528897024/build-log.txt

Reply:
I think you could pass add a new var(`enable-blobfuse-proxy`) in helm chart, and then if `ENABLE_BLOBFUSE_PROXY` is defined, then set `enable-blobfuse-proxy` as true:

```
.PHONY: e2e-bootstrap
e2e-bootstrap: install-helm
	# Only build and push the image if it does not exist in the registry
	docker pull $(IMAGE_TAG) || make blob-container push
	helm install blob-csi-driver ./charts/latest/blob-csi-driver --namespace kube-system --wait --timeout=15m -v=5 --debug \
		--set controller.runOnMaster=true \
		--set controller.replicas=1 \
		--set cloud=$(CLOUD) \
		$(E2E_HELM_OPTIONS)
```